### PR TITLE
added_upload_watermark_option

### PR DESCRIPTION
**Pull request by Dysfunctional89**

### CODE REVIEW

File: src/components/Watermarks/UploadWatermarks.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
+import React, {CSSProperties, useState} from 'react';
```
Comment (zlayabekrija) [inline src/components/Watermarks/UploadWatermarks.tsx:1]: Please reuse Upload step one component as much as possible.

Please reach out to @MateoKusan98  or @BaneIT

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Please apply requested changes

File: src/components/Upload/Upload.tsx [hunk 2]
```diff
@@ -117,12 +135,15 @@ export default function UploadStepOne() {
           <aside style={thumbsContainer as CSSProperties}>{thumbs}</aside>
         </div>
 
-        <button className={`proceed ${toggleProceed}`} type="submit" onClick={handleProceed}>
-          Proceed
+        <button className={`proceed ${toggleProceed}`} type="submit" onClick={() => submit ? handleSubmit() : handleProceed()}>
```
Comment (lazarevic-00) [inline src/components/Upload/Upload.tsx:138]: You should pass only one function if you are trying to create reusable component.

File: src/components/UploadImages/UploadStepOne.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import {imagesApi} from '../../api';
+import {imagesSlice} from '../../store/slices/images.slice';
+import Upload from '../Upload/Upload';
+
+export default function UploadStepOne() {
+   
+  
+  
+  return (
+    <>
```
Comment (lazarevic-00) [inline src/components/UploadImages/UploadStepOne.tsx:11]: You don't need fragment here, also, API should be already assigned here.

File: src/components/Watermarks/UploadWatermarks.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React  from 'react';
+import {watermarksApi} from '../../api';
+import {watermarkSlice} from '../../store/slices/watermark.slice';
+import Upload from '../Upload/Upload';
+
+export default function UploadWatermarks(isDefault:any) {
+  
+  return (
+        <>
```
Comment (lazarevic-00) [inline src/components/Watermarks/UploadWatermarks.tsx:9]: Same here

File: src/components/Watermarks/Watermarks.tsx [hunk 5]
```diff
@@ -30,11 +30,7 @@ export default function Watermarks({isDefault}: IWatermarksProps) {
 
   return (
     <div>
-      <div className="d-flex gap-5 ms-5 mt-">
-        {data?.map((watermark) => (
-          <Watermark key={watermark.name} {...watermark} user={user} />
-        ))}
-      </div>
+      <div className="row">{data && data?.map((watermark) => <Watermark key={watermark.name} {...watermark} user={user} />)}</div>
```
Comment (lazarevic-00) [inline src/components/Watermarks/Watermarks.tsx:33]: You dont need here "?" on data if we have data &&

**Review by lazarevic-00 — CHANGES_REQUESTED** [review]
Please apply suggested changes

File: src/Pages/AdminPage/AdminPageUploadWatermarks.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {useNavigate} from 'react-router-dom';
+import {watermarksApi} from '../../api';
+import Upload from '../../components/Upload/Upload';
+import {RootState} from '../../store';
+import {watermarkSlice} from '../../store/slices/watermark.slice';
+
+export default function AdminPageUploadWatermarks(isDefault: any) {
```
Comment (BaneIT) [inline src/Pages/AdminPage/AdminPageUploadWatermarks.tsx:9]: Remove the parameters if you don't use them

**Review by zlayabekrija — APPROVED** [review]
Great work :+1:

**Pull request merged**